tests/containers/buildah_docker.pm

```diff
@@ -46,7 +46,7 @@ sub run {
                 test_opensuse_based_image(image => "${prefix_img_name}-working-container", runtime => 'buildah', version => $version);
                 # Due to the steps from the test_opensuse_based_image previously,
                 # the image has been committed as refreshed
-                test_containered_app(runtime => 'docker', buildah => 1, dockerfile => 'Dockerfile.suse', base => 'refreshed') unless is_sle('<15', $version);
+                build_and_run_image(runtime => 'docker', buildah => 1, dockerfile => 'Dockerfile', base => 'refreshed');
```

Comment:
i guess you can make `Dockerfile` the default now and avoid to pass it every time. but it is up to you

Reply:
Please re-check 😄